main/inc/lib/template.lib.php

```diff
@@ -1075,11 +1075,67 @@ public function handleLoginFailed()
         return Display::return_message($message, 'error', false);
     }
 
+    public static function displayCASLoginButton($label = null)
+    {
+        $course = api_get_course_id();
+        $form = new FormValidator(
+            'form-cas-login',
+            'POST',
+            $course
+                ? api_get_path(WEB_COURSE_PATH).$course.'/'
+                : api_get_path(WEB_PATH).api_get_setting('page_after_login'),
+            null,
+            null,
+            FormValidator::LAYOUT_BOX_NO_LABEL
+        );
+        $form->addHidden('forceCASAuthentication', 1);
+        $form->addButton(
+            'casLoginButton',
+            is_null($label) ? sprintf(get_lang('LoginWithYourAccount'), api_get_setting("Institution")) : $label,
+            api_get_setting("casLogoURL"),
+            'primary',
+            null,
+            'btn-block'
+        );
+
+        return $form->returnForm();
+    }
+
+    public static function displayCASLogoutButton($label = null)
+    {
+        $form = new FormValidator(
+            'form-cas-logout',
+            'GET',
+            api_get_path(WEB_PATH),
+            null,
+            null,
+            FormValidator::LAYOUT_BOX_NO_LABEL
+        );
+        $form->addHidden('logout', 1);
+        $form->addButton(
+            'casLogoutButton',
+            is_null($label) ? sprintf(get_lang('LogoutWithYourAccount'), api_get_setting("Institution")) : $label,
+            api_get_setting("casLogoURL"),
+            'primary',
+            null,
+            'btn-block'
+        );
+
+        return $form->returnForm();
+    }
+
     /**
      * @return string
+     * @throws Exception
      */
-    public function displayLoginForm()
+    public static function displayLoginForm()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The variable $captcha_question is not named in camelCase.](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=4906847)